Python/test.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import os
+
+output = input("Enter your name: ")
+print("Your name is " + output)
+
+
+arr = np.array([1, 2, 3, 4])
+
+print(arr[1]) 
+
+print(os.getenv("USER"))
```

Comment:
Use the ENV variable that is passed through Docker File

Reply:
DONE